### DISCUSSION TITLE
Browser notification

### DISCUSSION THREAD

**Discussion by Mikerhinos** [Comfy-Org/ComfyUI#950] [Ideas]
It was probably already asked (didn't found it with searching for "notification" though), but would it be possible to have a browser notification when an image is done loading in the preview and save image node ?

**Comment by reaper47** [comment]
There is this Favicon Status extension you could try: https://github.com/pythongosssss/ComfyUI-Custom-Scripts.

**Comment by barepixels** [comment]
playing a mp3 file would be awesome

**Reply by tusharbhutt** [reply]
There is... https://github.com/pythongosssss/ComfyUI-Custom-Scripts#play-sound
